Commands/Config/PresetCommand.cs

```diff
@@ -0,0 +1,117 @@
+using ArtcordBot.Helpers;
+using DSharpPlus.Commands;
+using DSharpPlus.Commands.Processors.SlashCommands.ArgumentModifiers;
+using DSharpPlus.Entities;
+
+
+namespace ArtcordBot.Features.ConfigCommands
+{
+    public partial class ConfigCommandsGroup
```

Comment:
```suggestion
    [Command("preset")]
    public partial class ConfigCommandsGroup
```

All of these commands should be under `/preset`, so instead of it being `/addPreset` it should be `/presets add`. If you want you can also name them macros or whatever if you think presets is a confusing name